config/default.json5

```diff
@@ -0,0 +1,85 @@
+        ERROR: 4;
+        SILENT: 5;
+      }
+    */,
+    workdir: '.', // path to the folder where the server will store the database and all its data.
+    readyTimeout: 30000,
+    checkInterval: 10000,
+    disableSponsoredTx: false,
+    feePercentage: 0 /* allows revenue sharing feature and sets the fee value (%) that the worker will take from all transactions.
+    - the fee will be added to the estimated gas and required in the transaction amount.
+    - the percentage is represented as a fraction (1 = 100%) string to allow for very low or high percentages
+    - the minus sign is omitted if used
+    - fractions exceeding the number of decimals of that of the native currency will be rounded up
+   */,
+    sponsoredDestinations: [],
+    requestMinValidSeconds: 43200,
+  },
+  /*
+    Blockchain node
+  */
+  blockchain: {
+    rskNodeUrl: 'http://127.0.0.1:4444', //  RSK node endpoint URL, where the RSK node is located.
+    gasPriceFactor: 1,
+    alertedBlockDelay: 0,
+    minAlertedDelayMS: 0,
+    maxAlertedDelayMS: 0,
+    registrationBlockRate: 0,
+    workerMinBalance: 0.001e18, // 0.001 RBTC
+    workerTargetBalance: 0.003e18, // 0.003 RBTC
+    managerMinBalance: 0.001e18, // 0.001 RBTC
+    managerMinStake: 1, // 1 wei
+    managerTargetBalance: 0.003e18, // 0.003 RBTC
+    minHubWithdrawalBalance: 0.001e18, // 0.001 RBTC
+    refreshStateTimeoutBlocks: 5,
+    pendingTransactionTimeoutBlocks: 30, // around 5 minutes with 10 seconds block times.
+    successfulRoundsForReady: 3, // successful mined blocks to become ready after exception.
+    confirmationsNeeded: 12,
+    retryGasPriceFactor: 1.2, // gas price factor used to calculate the gas on the server, you can leave it as 1.
+    defaultGasLimit: 500000,
+    maxGasPrice: 100000000000,
+    estimateGasFactor: 1.2,
+    versionRegistryDelayPeriod: 0,
+  },
+  /*
+    Relay contracts addresses
+  */
+  contracts: {
+    relayHubAddress: '0x0000000000000000000000000000000000000000', // relay hub contract address, you can retrieve this from the contract summary.
+    relayVerifierAddress: '0x0000000000000000000000000000000000000000', // relay verifier contract address, you can retrieve this from the contract summary.
+    deployVerifierAddress: '0x0000000000000000000000000000000000000000', // deploy verifier contract address, you can retrieve this from the contract summary.
+    versionRegistryAddress: '0x0000000000000000000000000000000000000000',
+    feesReceiver: '0x0000000000000000000000000000000000000000',
+    trustedVerifiers: [],
+    relayHubId: '',
+  },
+  register: {
+    stake: '0.01', // amount of stake to set up
+    funds: '0.02', // amount of funds to set up
+    mnemonic: '', // mnemonic to use for unlocking the account parameter
+    privateKey: '', // private key to retrieve the account address from
+    signer: {}, // Signer object (only when used as a library)
+    hub: '',
+    gasPrice: 60000000,
+    relayUrl: '',
+    unstakeDelay: 1000,
+  },
+}
```

Comment:
we do, the `config`, package get the configuration from the root folder, so it will be using this files instead of the server one, unless we configure over here to use the ones that inside the server.